_posts/2020-07-06-staying-alive-re-thinking-business-strategy-for-indonesian-msmes.markdown

```diff
@@ -1,18 +1,20 @@
+The COVID-19 pandemic is causing the largest global socio-economic disruption in decades. While  the micro, small and medium enterprises (MSMEs) were hailed as the heroes in the aftermath of 2008 global financial crises and had helped cushion Indonesia’s economy, today they offer less of a buffer given that the public health crisis has generated both supply and demand shocks. MSMEs are more vulnerable today as the pandemic is impacting most of the 64.2 million MSMEs in Indonesia. MSMEs play a crucial role in Indonesia's economy and absorb about 97 percent of the total national workforce while account for more than half of the country’s GDP. Therefore, how the MSMEs population can navigate this economic downturn will determine the extent to which jobs and productivity in Indonesia can be preserved.  The Indonesian government has been struggling with how to manage informal MSMEs. According to Statistics Indonesia, about 98 percent of the enterprises operating in Indonesia are micro in size, with small enterprises only representing about 1.2 percent and medium enterprises representing even less around 0.09 percent of total enterprises(1). Despite a decade of business registration and licensing simplification reforms in Indonesia to make it easier and cheaper for enterprises to formalize, Statistics Indonesia still found that 93 percent of the micro and small enterprises operating in the non-agricultural sector in 2018 remained informal. The reforms also did not reduce informal employment, given that in 2018 an estimated 70 percent of the informal workers in Indonesia are employed by micro and small enterprises.
```

Comment:
Does the template parser translate the single quotes here automatically or should we be using HTML entities?

Reply:
I'm pretty sure it translates them.  I'll double check locally.